mlflow/models/evaluation/default_evaluator.py

```diff
@@ -79,7 +80,9 @@ def _infer_model_type_by_labels(labels):
 def _extract_raw_model(model):
     model_loader_module = model.metadata.flavors["python_function"]["loader_module"]
     if model_loader_module == "mlflow.sklearn" and not isinstance(model, _ServedPyFuncModel):
-        return model_loader_module, model._model_impl.sklearn_model
+        if isinstance(model._model_impl, _SklearnModelWrapper):
+            return model_loader_module, model._model_impl.sklearn_model
+        return model_loader_module, model._model_impl
```

Comment:
What's the case it is not `_SklearnModelWrapper` instance ? sklearn model._model_impl should be returned by sklearn flavor `_load_pyfunc`

Reply:
There's a test case in universe/automl that directly construct a PyFuncModel by passing _model_impl which uses sklearn Pipeline model.

Reply:
emm... is it a legal usage ?

Reply:
I guess other flavor might have similar issue if we directly construct `_model_impl`, but `_model_impl` is internal attribute managed by pyfunc if I understand it correctly

Reply:
I think if someone want to customize `_model_impl` attribute in sklearn pyfunc model,

then it should not be regarded as sklearn pyfunc model but he can create a custom pyfunc model that inherits `PythonModel` base class.